### DISCUSSION TITLE
Build config doesn't pick up 'destination' from jekyll's _config.yml

### DISCUSSION THREAD

**Discussion by aloukissas** [vercel/vercel#4356] [General]
What the title says. We had to override it in the vercel UI config with the same value.

It would be great if we could drive deployments with 100% version-controlled files (like _config.yml) and not have to manually enter overrides in the UI.

**Comment by aloukissas** [comment]
Note that the UI seems to imply that it will pick up config from _config.yml:

![image](https://user-images.githubusercontent.com/12943502/81735462-eee91e80-9449-11ea-80a1-5621209bd110.png)

**Comment by cblage** [comment]
In addition to this, it also doesn't seem to be respecting Jekyll's `baseurl` config, we have this in the `_config.yml`:

```
baseurl: "/blog" # the subpath of your site, e.g. /blog
url: "https://banana.com" # the base hostname & protocol for your site, e.g. http://example.com
```

**Comment by styfle** [comment]
Hi @alepacheco 

I created a new project using the [jekyll example](https://github.com/vercel/vercel/tree/master/examples/jekyll) and then added a line `destination: output` to `_config.yml` and it worked correctly with default project settings in vercel.

![image](https://user-images.githubusercontent.com/229881/83188485-c1fd6280-a0fd-11ea-8a3d-5bfa9d9decfb.png)
